metadata/install-qa-check.d/60cargo-eclass

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# QA check: ensure cargo based ebuilds define correct RUST_MIN_VER
+# Maintainer: Rust Project <rust@gentoo.org>
+
+cargo_ver_check() {
+	has cargo ${INHERITED} || return
+
+	# First version of Rust to support that edition
+	declare -A RUST_EDITIONS=(
+		[2024]="1.85.0"
+		#[2021]="1.56.0" # our oldest rust version is 1.74.0
+	)
+
+	# Maximum value for "edition" across Cargo.toml files
+	local cargo_toml_edition=$(
+		grep --recursive "${S}" --include="Cargo.toml" -E -e '^\s*edition\s*=\s*"' | \
+		cut --delimiter=\" --fields=2 | \
+		sort --numeric-sort | \
+		tail --lines=1
+	)
+	if ((cargo_toml_edition)); then
```

Comment:
What does this check mean?

Reply:
didn't like this check, replaced with more readable one.